cmd/safetyculture-exporter/cmd/root.go

```diff
@@ -90,6 +90,10 @@ func configFlags() {
 	// TODO - Can we validate these tokens and throw error if they are wrong?
 	connectionFlags.StringP("access-token", "t", "", "API Access Token")
 	connectionFlags.String("api-url", "https://api.safetyculture.io", "API URL")
+	connectionFlags.StringP("sheqsy-username", "", "", "SHEQSY API Username")
+	connectionFlags.StringP("sheqsy-password", "", "", "SHEQSY API Password")
```

Comment:
I'm very on the edge about flat out declining this PR solely based on the fact that we have to use username/password auth.

Reply:
From my testing I don't think this is actually the username and password of an organisation. These are actually an key/secret pair for the company in SHEQSY.

Using my username and password did not work.